package.json

```diff
@@ -46,8 +46,8 @@
     "chalk": "^2.3.2",
     "classnames": "^2.2.5",
     "cookie-parser": "^1.4.3",
+    "cookie-session": "^2.0.0-beta.3",
```

Comment:
`-beta.3` is their latest version for about a year.

<img width="795" alt="screen shot 2018-10-01 at 1 27 41 pm" src="https://user-images.githubusercontent.com/5229321/46291584-6a040680-c57e-11e8-9475-6a0fd68cb76a.png">

I'm not sure if to stay on this version or use a previous. Or maybe consider another package? suggestions welcomed?

Reply:
I also saw some errors in Snyk. I haven't looked well, but it looks like they are from nextjs and not the addition of this package.